federation/apis/federation/deep_copy_generated.go

```diff
@@ -23,108 +23,133 @@ package federation
 import (
 	api "k8s.io/kubernetes/pkg/api"
 	conversion "k8s.io/kubernetes/pkg/conversion"
+	reflect "reflect"
 )
 
 func init() {
 	if err := api.Scheme.AddGeneratedDeepCopyFuncs(
-		DeepCopy_federation_Cluster,
-		DeepCopy_federation_ClusterCondition,
-		DeepCopy_federation_ClusterList,
-		DeepCopy_federation_ClusterSpec,
-		DeepCopy_federation_ClusterStatus,
-		DeepCopy_federation_ServerAddressByClientCIDR,
+		conversion.GeneratedDeepCopyFunc{Fn: DeepCopy_federation_Cluster, InType: reflect.TypeOf(func() *Cluster { var x *Cluster; return x }())},
```

Comment:
document it?

On Tue, Aug 9, 2016 at 5:57 PM, Daniel Smith <notifications@github.com>
wrote:

> In federation/apis/federation/deep_copy_generated.go
> <https://github.com/kubernetes/kubernetes/pull/28679#discussion_r74170915>
> :
>
> >  )
> >
> >  func init() {
> >  	if err := api.Scheme.AddGeneratedDeepCopyFuncs(
> > -		DeepCopy_federation_Cluster,
> > -		DeepCopy_federation_ClusterCondition,
> > -		DeepCopy_federation_ClusterList,
> > -		DeepCopy_federation_ClusterSpec,
> > -		DeepCopy_federation_ClusterStatus,
> > -		DeepCopy_federation_ServerAddressByClientCIDR,
> > +		conversion.GeneratedDeepCopyFunc{Fn: DeepCopy_federation_Cluster, InType: reflect.TypeOf(func() *Cluster { var x *Cluster; return x }())},
>
> Hm, I see why you did it this way.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/28679/files/629b25daf93e55464f5721f0b06e3ed3c6a48552#r74170915>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFVgVHk0rz5LPb7lgpqPrGXn_tx32l6Xks5qeSH9gaJpZM4JH-mq>
> .
>